reco_utils/setup.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from setuptools import setup, find_packages
+from os import path
+
+
+VERSION = __import__('__init__').VERSION
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='reco_utils',
```

Comment:
minor thing, but it would be good to apply black

Reply:
yep, will do

Reply:
hey @gramhagen, I saw that you commited that black was applied, but the changes were not reflected in the files

Reply:
oh weird. doing it again (for the first time?)